packages/core/src/cloud/utils.ts

```diff
@@ -0,0 +1,18 @@
+import { PlatformApiClient } from "@llamaindex/cloud";
```

Comment:
NOTE: As we're using `await import` below, this import won't translate to a `require` call (it's just used for type checking by Typescript)